crates/typed-store/src/metrics.rs

```diff
@@ -22,6 +22,17 @@ const LATENCY_SEC_BUCKETS: &[f64] = &[
     0.001, 0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1., 2.5, 5., 10., 20., 30., 60., 90.,
 ];
 
+const BYTES_BUCKETS: &[f64] = &[
+    1.,
```

Comment:
I think using finer grained buckets can work too, e.g. `exponential_buckets(1.0, 4.0, 15)`

Reply:
Cool. Thanks for the tip. Switched to use exponential_buckets